### PR TITLE
Restructured theme docs

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Fixed tickets | none
| License | MIT

#### What's in this PR?

This bundle changes the documentation to fit the removed liip-theme bundle and adds the documentation to install the sulu/theme-bundle.

#### Why?

The theming for sulu was extracted into an own bundle.

### CODE REVIEW

File: book/creating-a-basic-website/adding-a-theme.rst [hunk 1]
```diff
@@ -18,6 +18,62 @@ decide which theme to use, by a simple key in the webspace configuration file
...
+        load_controllers: false
+        assetic_integration: true
+
+This will configure a default theme which can be enabled in the
+`app/Resources/webspaces/<webspace>.xml` file by adding:
+
+.. code-block:: xml
+
+    <theme>
+        <key>default</key>
```
Comment (danrot) [inline book/creating-a-basic-website/adding-a-theme.rst]: The `key` does not exist anymore.

File: book/creating-a-basic-website/configuring-image-formats.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
+Configure image formats
+=======================
+
+If you are using images, you probably also care about the available image
+formats. Sulu also supports you with that issue. You can define different image
+formats, which Sulu will then create for every uploaded image. This generation
```
Comment (danrot) [inline book/creating-a-basic-website/configuring-image-formats.rst]: remove `then`

File: book/creating-a-basic-website/configuring-image-formats.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+is based on your configuration. You can define the formats in the file
+`Resources/themes/<theme>/config/image-formats.xml`. Take a look at the
```
Comment (danrot) [inline book/creating-a-basic-website/configuring-image-formats.rst]: Theme specific, should also list the variant without themes installed.

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 4]
```diff
@@ -40,7 +40,12 @@ In :doc:`adding-a-template` we learned how to define a template.
     </template>
 
 
-In the page template the view could be set.
+In the page template the view could be set. Internally sulu appends the format
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: `Sulu`

File: book/creating-a-basic-website/index.rst [hunk 5]
```diff
@@ -13,10 +13,11 @@ into smart content and localization.
...
     adding-a-template
-    adding-a-theme
+    configuring-image-formats
     creating-a-twig-template
     using-smart-content
     adding-localizations
+    adding-a-theme
 
 At first we'll have another look at the content architecture on a technical level
-and afterwards we will create stuff.
```
Comment (danrot) [inline book/creating-a-basic-website/index.rst]: Please add a new line

File: book/creating-a-basic-website/setup-a-webspace.rst [hunk 6]
```diff
@@ -101,12 +98,14 @@ the available localizations:
...
 
-Themes
-------
+Themes (optional)
+-----------------
 
 The `theme` is described by a key. This key leads to a certain theme,
 implemented by a developer in the system. Read more about themes in the section
-:doc:`adding-a-theme`.
+:doc:`adding-a-theme`. This feature is default deactivated and therfor in the
```
Comment (danrot) [inline book/creating-a-basic-website/setup-a-webspace.rst]: `therefore`

File: book/creating-a-basic-website/index.rst (hunk 5 above)
Comment (wachterjohannes) [inline book/creating-a-basic-website/index.rst]: there is already one but i think this was not there before.

**Comment by wachterjohannes** [comment]
@danrot this should be fixed now.

**Pull request merged**